### PR TITLE
Create C# sample for Azure Sentinel API

### PR DESCRIPTION
**Pull request by chinguyen1**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by preetikr** [comment]
@chinguyen1 - Please remove the stats.md file from your PR. That's an auto generated file and should not be included in the PR.

**Review by preetikr — CHANGES_REQUESTED** [review]
Did you receive feedback from feature teams on a go on this change? Would be great to close on this and merge.

**Comment by chinguyen1** [comment]
@preetikr Kobi's team (Actions API) has reviewed his part of API & all looks good. I'm still waiting on other teams to review their parts. But these parts are essentially similar to Actions API. If you're ok with it, we can merge this, then if users have any questions, they can raise issues & I will help take care of those. I opened a new PR with some more fixes: https://github.com/Azure/Azure-Sentinel/pull/1274

**Pull request closed without merging**
